packages/frontend/src/components/sign/SignTransferRetry.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import ArrowUpImage from '../../images/icon-arrow-up-green.svg';
+import RetryImage from '../../images/icon-retry-tx.svg';
+import FormButton from '../common/FormButton';
+
+const CustomContainer = styled.div`
+    max-width: 450px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: #25282A;
+    text-align: center;
+
+    .title {
+        margin-top: 24px;
+
+        h2 {
+            font-weight: 900;
+            font-size: 22px;
+            color: #24272a;
+        }
+    }
+
+    && .text {
+        color: #72727A;
+        margin-top: 24px;
+    }
+
+    .buttons {
+        display: flex;
+        width: 100%;
+        margin-top: 40px;
+
+        button {
+            flex: 1;
+
+            &:last-of-type {
+                margin-left: 30px !important;
+
+                @media (min-width: 768px) {
+                    margin-left: 50px !important;
+                }
+            }
+        }
+    }
+
+    .fees {
+        width: 100%;
+        border: 1px solid #F0F0F1;
+        padding: 15px;
+        border-radius: 8px;
+        margin-top: 30px;
+        color: #72727A;
+
+        b {
+            color: #25282A;
+        }
+
+        .fees-line {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            margin-top: 12px;
+
+            .tgas {
+                color: #00C08B;
+                position: relative;
+
+                :after {
+                    content: '';
+                    position: absolute;
+                    background: url(${ArrowUpImage}) center top no-repeat;
+                    width: 16px;
+                    height: 17px;
+                    left: -22px;
+                    top: 1px;
+                }
+            }
+        }
+    }
+`;
+
+// TODO: Why handleDeny? It's not an error.
```

Comment:
Maybe just rename it within this context to `handleCancel`? Even though you can still re-use the `handleDeny` function to send user back to the app

Reply:
It's not my comment, I copied it with other code from `SignTransferSuccess.js` :)

Reply:
Even more of a reason to name it correctly in your component; don't let things like this become infectious

Reply:
Sure, I just prefer to keep the same name for one method across components for one page (we have a `Deny` button on first screen, `Cancel` button on the retry tx screen, we also `Close` button on the cancel page, all of them with the same method :) ). Maybe call this method `handleCancel` everywhere (I did for now)?

Reply:
handleCancel sounds reasonable for the local scope inside each component :+1: